test/utils/adjustMarketPrice.spec.ts

```diff
@@ -0,0 +1,194 @@
+import { checkMarketAndSmartAdjust } from 'hooks/usePrioritiseTokensForPrice'
+import tokenList from '../data/tokenList'
+
+// QUOTE gives price
+//     BASE/QUOTE
+// e.g WETH/TUSD
+
+describe('Check market and check base/quote token priorities', () => {
+  const [WETH, TUSD, USDT, , , , , FAKE] = tokenList
+
+  test('STABLE/STABLE => return with no change', () => {
```

Comment:
Ubernit: you can create inner "describe" and group by "STABLE/STABLE", then you can do the inner tests for this two.

USDT/USDC  --> USDT/USDC
USDC/USDT  --> USDC/USDT

not ubernit: I think you haven't tested the second case above ^^^.